.circleci/config.yml

```diff
@@ -30,3 +30,11 @@ jobs:
             - store_artifacts:
                 path: doc/_build/html/
                 destination: html
+            - deploy:
+                name: github pages deployment
+                command: |
```

Comment:
you should do it only for `master` branch with CIRCLE_BRANCH command. Then I think once I merge, it should work ... not sure but I think it should

Reply:
so reading up on this, CIRCLE_BRANCH is an environment variable pointing to the branch that is currently being built. I am not really sure what you mean by "I should use it" ... we still want to push the built docs against our `gh-pages` branch, don't we?

Reply:
The deploy should be done only for `master` branch, not for the PR branches. And the pushing to `gh-pages` is taken care of by `make install`

Reply:
Ah I see ... okay - fingers crossed then ;-)